src/serializer.js

```diff
@@ -34,20 +34,20 @@ var Serializer = function () {
   /* pass */
   }
 
-  __Serializer.prototype.setBase = function (base) { this.base = base ; return this}
+  __Serializer.prototype.setBase = function (base) { this.base = base; return this }
```

Comment:
I was trying to remember whether it was python or coffee - -maybe this was when coffee  was the thing... .. as I don't see any pyjslib calls.